docs/modules/ROOT/pages/cops_rails.adoc

```diff
@@ -725,16 +725,16 @@ end
 
 Prevents usage of `"*"` on an Arel::Table column reference.
 
-Using `arel_table["*"]` causes the outputted string to be a literal
-quoted asterisk (e.g. <tt>`my_model`.`*`</tt>). This causes the
-database to look for a column named <tt>`*`</tt> (or `"*"`) as opposed
+Using `arel_table["\*"]` causes the outputted string to be a literal
+quoted asterisk (e.g. `my_model`.`*`). This causes the
+database to look for a column named `\*` (or `"*"`) as opposed
 to expanding the column list as one would likely expect.
 
 [#safety-railsarelstar]
 === Safety
 
-This cop's autocorrection is unsafe because it turns a quoted `*` into
-an SQL `*`, unquoted. `*` is a valid column name in certain databases
+This cop's autocorrection is unsafe because it turns a quoted `\*` into
+an SQL `*`, unquoted. `\*` is a valid column name in certain databases
```

Comment:
As indicated at the top of the file, this document is auto-generated and does not need to be updated manually.
https://raw.githubusercontent.com/rubocop/rubocop-rails/refs/tags/v2.30.3/docs/modules/ROOT/pages/cops_rails.adoc

Reply:
Oh, I overlooked the notice at the beginning of the file. Sorry about that! I've made the corrections again.
https://github.com/rubocop/rubocop-rails/pull/1469/commits/c2227561e49f1156ca82104f120415d302a6fbe8